File: app/src/main/java/seigneur/gauvain/mycourt/utils/timber/TimberReleaseTree.java
package seigneur.gauvain.mycourt.utils.timber;
import timber.log.Timber;


public class TimberReleaseTree extends Timber.Tree {
    @Override
    protected void log(int priority, String tag, String message, Throwable t) {
        //Use CrashLytics...
        /*if (priority == ERROR || priority == WARNING) {
            //Crashlytics.log(int priority, String tag, String msg);
        } else {
            //bitch please, is nothing!
        }*/

    }
}

File: app/src/main/java/seigneur/gauvain/mycourt/utils/timber/TimberReleaseTree.kt
package seigneur.gauvain.mycourt.utils.timber

import timber.log.Timber

class TimberReleaseTree : Timber.Tree() {
    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
        //Use CrashLytics...
        /*if (priority == ERROR || priority == WARNING) {
            //Crashlytics.log(int priority, String tag, String msg);
        } else {
            //bitch please, is nothing!
        }*/

    }
}